### ISSUE TITLE
Which specifications/standards fulfil the requirements of the HVD Regulation for APIs?

### ISSUE THREAD

**Issue by hogredan** [INSPIRE-MIF/hvd-inspire#2]
In Article 2 (6) of the Regulation 2023/138 “Application programming interface (API)” is defined as “a set of functions, procedures, definitions and protocols for machine-to-machine communication and the seamless exchange of data”. In addition, the Regulation requires in Article 3 (2) and (3) a documentation and publication of the terms of use of the API.

The INSPIRE Directive and the implementing rules regarding network services define operations (comparable with the term “functions” used in the Regulation 2023/138) for downloading spatial data through download services. In the (publicly available) Technical Guidance document regarding download services these operations are specified by “definitions and protocols for machine-to-machine communication”. The terms of use of a download service are described in the service metadata.

Thus, in Germany, we consider that all specifications/standards described in the INSPIRE Technical Guidance and Good Practice documents fulfil the requirements of the HVD Regulation for APIs, including the implementation based on Atom and OpenSearch.

Basically, the underlying OGC/ISO interfaces also fulfil the requirements for APIs:
•	WFS (https://www.ogc.org/standard/wfs/)
•	OGC API-Features (https://ogcapi.ogc.org/features/)
•	WCS (https://www.ogc.org/standard/wcs/)
•	Sensor Things API (https://www.ogc.org/standard/sensorthings/)

Regarding the provision of high-value datasets that fall under an INSPIRE theme, the above-mentioned INSPIRE network services are to be used.

Regarding high-value datasets that do not fall under an INSPIRE theme, the requirements of the national SDI needs to be taken into account to achieve interoperability within Germany. The specifications of the German SDI are basically compatible with European and international standards.

Do the other Member States share this view?

**Comment by idevisser** [comment]
we see it slightly differently, see

https://docs.geostandaarden.nl/eu/hvd/englishfactsheets/230214FactsheetAPIs-HVDLv4_EN.pdf
